plugins/ginkgo/src/ginkgo_logger.h

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright 2020 IFPEN-CEA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ginkgo/ginkgo.hpp>
+#include <iomanip>
+
```

Comment:
Goal of this file ?
Can be moved in bench folder ?

Reply:
Replaced by a class with private attibutes, public accessor, and friend vector and matrix classes.